### ISSUE TITLE
Allow aws_acm_certificate to query the most recent certificate

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#546]
_This issue was originally opened by @fabian-dev as hashicorp/terraform#12111. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

By now using the `aws_acm_certificate` data source is no option for us. This is because we want to renew our ACM certs on a regular basis.

During our renewal phase and after issuing new certs, there will be more than one cert present (with equal domain names).
TF will then fail with:
```
data.aws_acm_certificate.xxx: Multiple certificates for domain "xxx" found in this region.
```

If I could use a most_recent=true flag (like it is with the `aws_ami` data source), I could gracefully apply the TF configuration during the renewal phase and switch to new certs.

I would appreciate your thought on this.

Cheers Fabian

**Comment by bflad** [comment]
Thanks for your patience here, everyone! The `most_recent` functionality is now in master and will release in v1.9.0 of the AWS provider. Happy Terraform'ing! 🎉

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.9.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#190-february-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
